docs/03-computation/02-configuration.mdx

```diff
@@ -1,14 +1,24 @@
-# How Privacy Computation Engine Works
+---
+title: Configuring your Privacy Computation Engine
+sidebar_position: 2
+tags: ['guide']
+---
 
-## Configuring your Privacy Computation Engine
+import WipWarning from '../_partials/_wip.mdx';
+
+<WipWarning repo="https://github.com/blindnet-io/privacy-computation-engine" />
 
 You must first configure the Privacy Computation Engine in order to make the most of it.
+
 It needs to know about:
+
 - General information about your Organization and its' data processing policies (Equivalent to a RoPA - Record of Processing Activities)
 - Mappings of your data structures to [PRIV](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md) Data Categories
 - Few choices you have to make about when to authenticate the user (before or after they make a Privacy Request), and when to deliver automatic responses and when to wait for validation
 
 To Configure your Privacy Computation Engine prepare the following information:
+
```

Comment:
This is a very preliminary explanation of what they'll need to configure it. It is not really an instruction on HOW they can configure it. 

We really need here something more hands-on, how to call the API to feed it with this information. 

While waiting for this to be written, it is appropriate to make a disclamer that this is work-in-progress and that concrete API instructions will be added soon.

Reply:
I'm working on a new document based on this one and the one @m4rk055 wrote (see https://github.com/blindnet-io/blindnet.dev/pull/35) to partially answer this.

In the meantime, I've indeed put a disclaimer at the top but will update its wording ASAP to better fit your comment.

Reply:
Addressed in https://github.com/blindnet-io/blindnet.dev/pull/40

> **Note**
>
> #40 only addressed _« make a disclaimer that this is work-in-progress and that concrete API instructions will be added soon. »_
>
> To address, _« more hands-on, how to call the API to feed it with this information »_, the API will first need to be developed and available. In the meantime, #35 will still address it, but only partially.